docs/examples/viz_markers.py

```diff
@@ -0,0 +1,52 @@
+"""
+======================================================================
+Fury Markers
+======================================================================
+
+This it's a example which shows how to use the marker actor.
+"""
+import numpy as np
+import vtk
+import vtk.util.numpy_support as vtknp
+from fury import actor, window, colormap as cmap
+n = 10000
+
+"""
```

Comment:
please, use  `#####` to follow the standard of the other examples. It will not work otherwise